src/library/dao/cuedao.cpp

```diff
@@ -147,7 +154,7 @@ bool CueDAO::saveCue(Cue* cue) {
         query.bindValue(":length", cue->getLength());
         query.bindValue(":hotcue", cue->getHotCue());
         query.bindValue(":label", cue->getLabel());
-        query.bindValue(":color", cue->getColor()->m_iId);
+        query.bindValue(":color", static_cast<quint32>(cue->getColor()));
```

Comment:
Always use the corresponding type alias mixxx::RgColor::code_t instead of built-in types. That's what it is supposed for.